neuralmonkey/tests/test_chrf.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3.5
+
+
+import unittest
+
+from neuralmonkey.evaluators.chrf import ChrFEvaluator
+
+
+CORPUS_DECODED = [
```

Comment:
Neni lepší to naimportovat z testu na BLEU?

Reply:
Jasne, to asi jo. Nebo respektive to mit uplne bokem; porovnavani s hypo vet s refe vetama delas u vice metrik